third_party/move/move-compiler-v2/src/inliner.rs

```diff
@@ -840,21 +844,41 @@ impl<'env, 'rewriter> InlinedRewriter<'env, 'rewriter> {
         self.shadow_stack.enter_scope(entering_vars);
     }
 
-    /// Check for and warn about Return inside a lambda
+    /// Check for and warn about Return inside a lambda.
+    /// Also check for Break or Continue inside a lambda and not inside a loop.
     fn check_for_return_in_lambda(env: &GlobalEnv, lambda_body: &Exp) {
```

Comment:
Consider renaming function, as it is doing more than originally intended.

Reply:
Done.